src/deploy/osp_deployer/settings/sample_xsp_profile.ini

```diff
@@ -271,6 +271,9 @@ sriov_vf_count=64
 # Set the following option to true/false for DVR
 dvr_enable=false
 
+# Set to true to enable Barbican
```

Comment:
Use the same verbiage as the option for DVR above and Octavia below.  Should say "Set the following option to true/false for Barbican